### PR TITLE
[#1689] Add 'RegenerateLanguages documentation' to the Release notes.

### PR DESCRIPTION
**Pull request by miklossy**
Signed-off-by: Tamas Miklossy <miklossy@itemis.de>

### CODE REVIEW

File: xtext-website/_posts/releasenotes/2020-06-02-version-2-22-0.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ As you might have recognized, the number of people contributing to Xtext on a re
 
 The [Xtext community website](https://www.eclipse.org/Xtext/community.html) and the [Xtext documentation website](https://www.eclipse.org/Xtext/documentation/index.html) have been updated. New Xtext-based projects have been added, some editor features and the Unit testing / UI testing capabilities have been documented and the embedded code snippets have been updated as well.
 
+Documentation about [regenerating the test- and production Xtext languages](https://github.com/eclipse/xtext/blob/master/dev-doc/RegenerateLanguages.md) was added to enable more people to contribute to Xtext.
```
Comment (cdietrich) [inline xtext-website/_posts/releasenotes/2020-06-02-version-2-22-0.md:19]: am not sure if this is relevant to end users at all

Comment (miklossy) [reply xtext-website/_posts/releasenotes/2020-06-02-version-2-22-0.md:19]: I think it is a good opportunity to make the Xtext users aware that such a `dev-doc `documentation exists and where it is located.

Comment (cdietrich) [reply xtext-website/_posts/releasenotes/2020-06-02-version-2-22-0.md:19]: maybe you can generalize this more (e.g. enhanced developer documentation at ....

Comment (miklossy) [reply xtext-website/_posts/releasenotes/2020-06-02-version-2-22-0.md]: Good idea! I updated the PR accordingly.

File: xtext-website/_posts/releasenotes/2020-06-02-version-2-22-0.md [hunk 2]
```diff
@@ -16,6 +16,8 @@ As you might have recognized, the number of people contributing to Xtext on a re
 
 The [Xtext community website](https://www.eclipse.org/Xtext/community.html) and the [Xtext documentation website](https://www.eclipse.org/Xtext/documentation/index.html) have been updated. New Xtext-based projects have been added, some editor features and the Unit testing / UI testing capabilities have been documented and the embedded code snippets have been updated as well.
 
+Furthermore, the [Xtext Developer Documentation](https://github.com/eclipse/xtext/tree/master/dev-doc) has been extended by a step-by-step introduction on [How to regenerate the test- and production Xtext languages](https://github.com/eclipse/xtext/blob/master/dev-doc/RegenerateLanguages.md) to enable the community to easily contribute to Xtext.
```
Comment (cdietrich) [inline xtext-website/_posts/releasenotes/2020-06-02-version-2-22-0.md:19]: have thought of more generalization here

Comment (miklossy) [reply xtext-website/_posts/releasenotes/2020-06-02-version-2-22-0.md]: @cdietrich I updated the PR as requested.

**Pull request merged**